### ISSUE TITLE
Bug NXDOMAIN

### ISSUE THREAD

**Issue by Nvbeh6** [tachiyomiorg/tachiyomi-extensions#11606]
### Source information

Doujindesu

### Steps to reproduce

Open Doujindesu Extension

### Expected behavior

Open Usually

### Actual behavior

Like This
![Screenshot_2022-04-25-20-01-58-703_eu kanade tachiyomi](https://user-images.githubusercontent.com/103346344/165094870-5320e13f-58d2-4b98-91bd-b5353625b651.jpg)

### Tachiyomi version

0.13.4

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by enjoe842** [comment]
> DoujinDesu and other Indonesian sites are being blocked by ISPs, resulting in a NXDOMAIN error or other errors. Solving this issue with DNS over HTTPS is not possible due to alternative DNS providers being blocked, DNS hijacking, etc. In order to keep using these sites, you must use a VPN or Deep Packet Inspection bypasser.

they changed the domain from https://doujindesu.xxx to https://212.32.226.234 maybe it's not an isp block problem

**Comment by alessandrojean** [comment]
This is not a new domain, it's a RAW IP. We don't use RAW IPs in the extensions, as it's a non-secure measure.

**Comment by enjoe842** [comment]
hm oke, for me it still working without a VPN, only using DNS in Android settings.
